### ISSUE TITLE
Button color icon different from the real color

### ISSUE THREAD

**Issue by jdzuri** [Clans/FloatingActionButton#47]
Hi,

I implement the first xml button example and the icon color into the button is different (gray) to the real icon color (white). I tried using a black icon but with the same result.

Thanks,

**Comment by chitgoks** [comment]
yes. what is the xml attribute that disables overriding the original color of the icon?
